### ISSUE TITLE
Setting the Right Incentives: Dynamic Scoring

### ISSUE THREAD

**Issue by blankster** [LAION-AI/Open-Assistant#1659]
The score you receive for a task can be a great motivator, but the current approach has led to situations where people only focus on the score and choose tasks that offer the highest points, which can have negative consequences.

In my opinion, the incentive system should be adjusted so that the highest points are given for completing tasks that lead to a "ready_for_export" conversation tree quickly and with good quality. Effort should also be considered, but the primary focus should be on the overall benefit to the project, and this should be rewarded with the most points.

Ideally, the scoring system should be dynamically adapting depending on where the most support is needed.

**Comment by 0x22almostEvil** [comment]
Yeah, we need more assistant's replies and labels rather than prompts.

**Comment by horribleCodes** [comment]
Good idea. The stat page helps, but I can only assume a lot of people don't really know in what state the trees are and how they can speed things along.

**Comment by andreaskoepf** [comment]
I have changed the weighting of initial prompts to 0.1: https://github.com/LAION-AI/Open-Assistant/commit/93d65bdbed73e154538453f8843274cbf416dce2 This change will become effective with the next deployment.

**Comment by andreaskoepf** [comment]
> The stat page helps, but I can only assume a lot of people don't really know in what state the trees are and how they can speed things along.

All available tasks beside writing initial prompts bring trees closer to the "ready for export" state. The most valuable contributions (also somewhat reflected by the score) is to write good assistant replies. I write "somewhat" because there are obviously people write very long and well researched replies and others users who don't invest the time. A really fair valuation of assistant replies is hard and would require AGI level intelligence (e.g. grading of essays).

**Comment by JohannesGaessler** [comment]
If we're talking purely about how maximizing score has the wrong incentives I think the bigger problem is that it disincentivizes thorough fact-checking. As far as I can tell you always get the same amount of score for labeling an assistant reply regardless of how much effort you put into determining the quality of the reply. So someone carefully checking every single claim in a reply will get a lower amount of score per hour than someone that just checks whether something _sounds_ correct and thus encourages the assistant to tell convincing lies if it can get away with it. I don't think there's an easy solution for this; I personally don't take the score that serious anyways.

There's also an issue in general where the amount of people that will be able to interact with your messages in a positive way will be much higher if the message is at the beginning of a tree rather than at the end. So this kind of disincentivizes putting in a lot of effort towards the end of a tree.

When it comes to balancing scores of different tasks against each other in particular we could maybe add additional leaderboards that only show the scores of some of the tasks rather than the full score? I'm not entirely sure how this would play out but maybe it will encourage users to get a better score in more than just one task; but I can also see how this might encourage users to focus even more on just one task since it would be easier to get at least one high rank by doing so.

**Comment by horribleCodes** [comment]
Maybe we shouldn't score writing assistant replies at all - instead, you should get rewarded for having your reply approved, I think we are already doing something like this, but we should lean more into it. There should be some back-and-forth between the writer trying to make their replies as factually accurate as possible, and the reviewer looking for mistakes as carefully as possible - this would encourage better data on both sides.

The tricky part is facilitating all this - if implemented poorly, it'd just incentivize reviewers to rate replies poorly. A decentralized solution would require a third party that inspects the classifications and downvotes them if they are unfair, but that just makes things more complex and pushes the potential for slacking to another location.
